starter-project-backend-g32/src/utils/ratingCRUD.js

```diff
@@ -0,0 +1,86 @@
+const getOne = (model) => async (req, res) => {
```

Comment:
file should be ts file, plus we don't need other folders, such as utils we can use controllers folder to handle business logics, clone master branch and work on that. plus we don't need crud for rating since its dependent on article you can add your function their.